setup.cfg

```diff
@@ -118,6 +118,8 @@ filterwarnings =
     # Warnings about open jp2 library not being found
     ignore:The openjp2 library:UserWarning
     ignore:Neither the openjp2 nor the openjpeg library could be loaded:UserWarning
+# Set the junit version to supress a depreaction warning
+junit_family=xunit1
```

Comment:
I was looking into this the other day and wondering if we actually want to bump it up rather than pin it down. Probably more a question for the azure templates though.